### ISSUE TITLE
Generated html file empty

### ISSUE THREAD

**Issue by dBenedek** [brentp/somalier#60]
Hello,

I have done the following steps:

1. extract sites:
```
for f in "$@"; do
  LOC="/disk/work/shared/data/MDS/${f}/alignment/STAR"
  cd ${LOC}
  for bam in *_Aligned.out_sorted_rg.bam; do
    base=$(basename $bam .bam);
    ${SOMALIER} extract -d ${OUT} --sites sites.hg38.rna.vcf.gz -f ${REF} ${LOC}/${bam}; 
    mv ${OUT}${f}.somalier ${OUT}${f}_${base}.somalier; 
  done
done
```

I renamed each file inside the for loop, because each dataset's bam files have the same read group, so the file names would be the same in case of the same dataset.

2. run somalier relate:
```
{SOMALIER} relate -g sample_groups.csv /data/somalier/*.somalier
```

After these steps, there's a .html output file (8.6 MB), however, it is blank, changing axes dows not affect anything.

What would cause this problem?

Thanks in advance,
Benedek Dankó

**Comment by brentp** [comment]
can you share the html file and the tsv output?

you could also try using the `--sample-prefix` argument to `somalier extract` so that each somalier file has a unique prefix.

**Comment by dBenedek** [comment]
Even if I use `--sample-prefix=$base` the generated .somalier files have the name from the bam files' read group tag.

Output files:
[somalier.samples.tsv.gz](https://github.com/brentp/somalier/files/5183871/somalier.samples.tsv.gz)
[somalier.html.gz](https://github.com/brentp/somalier/files/5183872/somalier.html.gz)

**Comment by brentp** [comment]
yes, you would change `--sample-prefix` to be unique for each sample, not to a shared $base.

**Comment by dBenedek** [comment]
> yes, you would change `--sample-prefix` to be unique for each sample, not to a shared $base.

OK, I will try that, thank you!

**Comment by brentp** [comment]
ok. please let me know if that solves the problem as that's why --sample-prefix exists.

**Comment by dBenedek** [comment]
Now the generated .tsv files look OK, but the html file is still blank. 

Output files:
[somalier.samples.tsv.gz](https://github.com/brentp/somalier/files/5185127/somalier.samples.tsv.gz)
[somalier.html.gz](https://github.com/brentp/somalier/files/5185128/somalier.html.gz)

**Comment by brentp** [comment]
ok. I'll make a fix for this for next release. you can edit somalier.html: find <script> and right after that, add a line that reads:
```
var inf = Infinity
```
and then save and reload.

**Comment by brentp** [comment]
The html doesn't look too informative. Usually RNA-Seq looks a bit better than this. Is there some reason these would be lower quality?
I'm still trying to improve for cases like this. If you get latest version (which still won't have the Infinity fix I described above), you can check a box that let's you view scaled IBS0 and IBS2 which might be more informative.

**Comment by dBenedek** [comment]
Thank you very much!

Yes, all samples are included, some of which have quite bad/lower quality.
